.github/workflows/core.yml

```diff
@@ -56,6 +56,7 @@ jobs:
           - python-version: '3.8'
             # only generate coverage report for a single python version in the matrix
             # to avoid overloading Codecov
+            # FIXME: reinstate coverage report before merging PR
```

Comment:
Should this be removed now?

Reply:
Thanks for catching that, I've removed it in 559b788.